# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/gtk-perl10/gtk-perl-build.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ur Gtk-Perl-0.7008/Applet/Makefile.PL Applet/Makefile.PL
--- Gtk-Perl-0.7008/Applet/Makefile.PL	Sat Mar 24 10:44:08 2001
+++ Gtk-Perl-0.7008/Applet/Makefile.PL	Mon Jul 22 10:20:27 2002
@@ -102,7 +102,7 @@
 sub MY::c_o {
 	package MY; # so that "SUPER" works right
 	my $inherited = shift->SUPER::c_o(@_);
-	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/m;
+	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/mg;
     $inherited;
 }
 
diff -ur Gtk-Perl-0.7008/Bonobo/Makefile.PL Bonobo/Makefile.PL
--- Gtk-Perl-0.7008/Bonobo/Makefile.PL	Wed May 30 08:39:05 2001
+++ Gtk-Perl-0.7008/Bonobo/Makefile.PL	Mon Jul 22 10:20:17 2002
@@ -96,7 +96,7 @@
 sub MY::c_o {
 	package MY; # so that "SUPER" works right
 	my $inherited = shift->SUPER::c_o(@_);
-	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/m;
+	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/mg;
     $inherited;
 }
 
diff -ur Gtk-Perl-0.7008/GdkImlib/Makefile.PL GdkImlib/Makefile.PL
--- Gtk-Perl-0.7008/GdkImlib/Makefile.PL	Fri Mar 23 08:57:07 2001
+++ Gtk-Perl-0.7008/GdkImlib/Makefile.PL	Mon Jul 22 10:20:40 2002
@@ -43,6 +43,6 @@
 sub MY::c_o {
 	package MY; # so that "SUPER" works right
 	my $inherited = shift->SUPER::c_o(@_);
-	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/m;
+	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/mg;
     $inherited;
 }
diff -ur Gtk-Perl-0.7008/GdkPixbuf/Makefile.PL GdkPixbuf/Makefile.PL
--- Gtk-Perl-0.7008/GdkPixbuf/Makefile.PL	Fri Mar 23 08:57:07 2001
+++ Gtk-Perl-0.7008/GdkPixbuf/Makefile.PL	Mon Jul 22 10:20:49 2002
@@ -89,6 +89,6 @@
 sub MY::c_o {
 	package MY; # so that "SUPER" works right
 	my $inherited = shift->SUPER::c_o(@_);
-	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/m;
+	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/mg;
     $inherited;
 }
diff -ur Gtk-Perl-0.7008/Glade/Makefile.PL Glade/Makefile.PL
--- Gtk-Perl-0.7008/Glade/Makefile.PL	Fri Mar 23 08:57:07 2001
+++ Gtk-Perl-0.7008/Glade/Makefile.PL	Mon Jul 22 10:20:31 2002
@@ -108,7 +108,7 @@
 sub MY::c_o {
 	package MY; # so that "SUPER" works right
 	my $inherited = shift->SUPER::c_o(@_);
-	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/m;
+	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/mg;
     $inherited;
 }
 
diff -ur Gtk-Perl-0.7008/Gnome/Makefile.PL Gnome/Makefile.PL
--- Gtk-Perl-0.7008/Gnome/Makefile.PL	Fri Mar 23 08:57:08 2001
+++ Gtk-Perl-0.7008/Gnome/Makefile.PL	Mon Jul 22 10:20:04 2002
@@ -125,7 +125,7 @@
 sub MY::c_o {
 	package MY; # so that "SUPER" works right
 	my $inherited = shift->SUPER::c_o(@_);
-	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/m;
+	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/mg;
     $inherited;
 }
 
diff -ur Gtk-Perl-0.7008/GnomePrint/Makefile.PL GnomePrint/Makefile.PL
--- Gtk-Perl-0.7008/GnomePrint/Makefile.PL	Fri Mar 23 08:57:08 2001
+++ Gtk-Perl-0.7008/GnomePrint/Makefile.PL	Mon Jul 22 10:20:45 2002
@@ -112,7 +112,7 @@
 sub MY::c_o {
 	package MY; # so that "SUPER" works right
 	my $inherited = shift->SUPER::c_o(@_);
-	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/m;
+	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/mg;
     $inherited;
 }
 
diff -ur Gtk-Perl-0.7008/Gtk/Makefile.PL Gtk/Makefile.PL
--- Gtk-Perl-0.7008/Gtk/Makefile.PL	Sat May 26 11:08:38 2001
+++ Gtk-Perl-0.7008/Gtk/Makefile.PL	Mon Jul 22 10:12:47 2002
@@ -190,7 +190,7 @@
 sub MY::c_o {
 	package MY; # so that "SUPER" works right
 	my $inherited = shift->SUPER::c_o(@_);
-	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/m;
+	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/mg;
     $inherited;
 }
 
diff -ur Gtk-Perl-0.7008/GtkGLArea/Makefile.PL GtkGLArea/Makefile.PL
--- Gtk-Perl-0.7008/GtkGLArea/Makefile.PL	Fri Mar 23 08:57:08 2001
+++ Gtk-Perl-0.7008/GtkGLArea/Makefile.PL	Mon Jul 22 10:20:01 2002
@@ -104,7 +104,7 @@
 sub MY::c_o {
 	package MY; # so that "SUPER" works right
 	my $inherited = shift->SUPER::c_o(@_);
-	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/m;
+	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/mg;
     $inherited;
 }
 
diff -ur Gtk-Perl-0.7008/GtkHTML/Makefile.PL GtkHTML/Makefile.PL
--- Gtk-Perl-0.7008/GtkHTML/Makefile.PL	Fri Mar 23 08:57:08 2001
+++ Gtk-Perl-0.7008/GtkHTML/Makefile.PL	Mon Jul 22 10:20:52 2002
@@ -107,7 +107,7 @@
 sub MY::c_o {
 	package MY; # so that "SUPER" works right
 	my $inherited = shift->SUPER::c_o(@_);
-	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/m;
+	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/mg;
     $inherited;
 }
 
diff -ur Gtk-Perl-0.7008/GtkXmHTML/Makefile.PL GtkXmHTML/Makefile.PL
--- Gtk-Perl-0.7008/GtkXmHTML/Makefile.PL	Fri Mar 23 08:57:08 2001
+++ Gtk-Perl-0.7008/GtkXmHTML/Makefile.PL	Mon Jul 22 10:20:36 2002
@@ -95,7 +95,7 @@
 sub MY::c_o {
 	package MY; # so that "SUPER" works right
 	my $inherited = shift->SUPER::c_o(@_);
-	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/m;
+	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/mg;
     $inherited;
 }
 
diff -ur Gtk-Perl-0.7008/Mozilla/Makefile.PL Mozilla/Makefile.PL
--- Gtk-Perl-0.7008/Mozilla/Makefile.PL	Fri Mar 23 08:57:08 2001
+++ Gtk-Perl-0.7008/Mozilla/Makefile.PL	Mon Jul 22 10:20:20 2002
@@ -94,7 +94,7 @@
 sub MY::c_o {
 	package MY; # so that "SUPER" works right
 	my $inherited = shift->SUPER::c_o(@_);
-	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/m;
+	$inherited =~ s/CCCMD.*$/$&\n\t\@if test -f `basename \$*.o` -a "`basename \$*.o`" != "\$*.o"; then mv `basename \$*.o` \$*.o; fi/mg;
     $inherited;
 }
 
